ui/src/pages/QuickSetup/AssignRolesPage.tsx

```diff
@@ -520,6 +518,21 @@ const issuerInstructions = () => {
   });
 };
 
+const getInstructionTriggers = (inst: InstructionType) => {
+  switch (inst) {
+    case InstructionType.INVESTOR:
+      return [MarketplaceTrigger.SettlementInstructionTrigger];
+    case InstructionType.CLEARING:
+    case InstructionType.CLEARINGHOUSE:
+    case InstructionType.MARKETCLEARING:
```

Comment:
@alexgraham-da can u confirm all of these need the clearing trigger?

Reply:
Only `InstructionType.CLEARINGHOUSE` needs the Clearing trigger, and I believe only `InstructionType.EXCHANGE` needs the settlement trigger.